app/api/custom/orders.py

```diff
@@ -93,3 +102,72 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
+    for attribute in attendee:
+        attendee[attribute.replace('-', '_')] = attendee.pop(attribute)
+    schema = AttendeeSchema()
+    json_api_attendee = {"data": {"attributes": data['attendee'], "type": "attendee"}}
+    result = schema.load(json_api_attendee)
+    if result.errors:
+        return jsonify(result.errors)
+    ticket_ids = [int(ticket['id']) for ticket in tickets]
+    ticket_info_list = db.session.query(Ticket).filter(Ticket.id.in_(ticket_ids)).filter_by(event_id=data['event_id'],
+                                                                                            deleted_at=None).all()
+    ticket_ids_found = [ticket_information.id for ticket_information in ticket_info_list]
+    tickets_not_found = list(set(ticket_ids) - set(ticket_ids_found))
+    if tickets_not_found:
+        return jsonify(status='Order Unsuccessful', error='Ticket with id {} was not found.'.format(tickets_not_found))
```

Comment:
There can be multiple tickets not found. Also, send event ID as well. Ticket 324 may exist but not under the event requested

Reply:
@iamareebjamal I am sending the list of all the tickets not found. Regarding example of ticket 324, I am already passing event_id as well which fetching the tickets, so those tickets will also throw tickets not found.

Reply:
It will throw: `Ticket with ID 324, 325, 326 was not found`

Everyone wants to see: `Tickets with ID 324, 325, 326 were not found in Event 432`

Hence, again https://github.com/fossasia/open-event-server/pull/6635#discussion_r356143003

Reply:
It will throw: `Ticket with ID 324, 325, 326 was not found in Event 432`

Everyone wants to see: `Tickets with ID 324, 325, 326 were not found in Event 432`